app/src/main/java/com/lubenard/oring_reminder/ui/EditEntryFragment.java

```diff
@@ -292,7 +292,7 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
                     } else {
                         if (Utils.checkDateInputSanity(formattedDatePut) == 1 && Utils.checkDateInputSanity(formattedDateRemoved) == 1) {
                             dbManager.updateDatesRing(entryId, formattedDatePut, formattedDateRemoved, 0);
-                            dbManager.endSession(entryId);
+                            dbManager.endSession(entryId,formattedDateRemoved);
```

Comment:
I think we can change ```endSession``` for ```endPause``` 
See here: 
https://github.com/lubenard/oRing_Reminder/blob/27dfbb5429f2fd8a97fbbbf72cd90f647ac126c5/app/src/main/java/com/lubenard/oring_reminder/DbManager.java#L264

No need to override the method, and it already only take a entryId as parameter.